functions/.env

```diff
@@ -1,2 +1,8 @@
 POSTFINANCE_EMAIL_USER=test@socialincome.org
 POSTFINANCE_EMAIL_PASSWORD=password
+
+TWILIO_SENDER_PHONE=+15005550006
+CLOUD_FUNCTIONS_URL=http://localhost:5001/social-income-prod/us-central1
+
+TWILIO_SID=ACac5dfe59a0af841c6b771835f331fabf
```

Comment:
Should this be committed? 😆 Do we need to reset the credentials and then add it to the .gitignore?